frontend-cert/js-projects/loan-qualification-checker/script.js

```diff
@@ -0,0 +1,84 @@
+const annualIncome = parseFloat(prompt('Enter your annual income:'));
+
+let creditScore;
```

Comment:
This causes an `illegal return statement` error.

At the end of the day, the `qualificationMessage` will resolve to `You do not qualify for any loans.` if they enter anything weird. Is that not enough?

Reply:
> This causes an illegal return statement error.

That's because there is no function. 
That return shouldn't be there.

Otherwise, I thought it would be nice to have them practice the logical OR operator.
But if you think the You do not qualify for any loans. message works then we don't have to include.

The only thing we would need to change is the `let creditScore;` 
It would need to be updated to the following so the logic is correct.

```js
const creditScore = parseFloat(prompt('Enter your credit score:'));
```